src/libraries/Microsoft.Extensions.Logging/tests/Common/LoggerMessageTest.cs

```diff
@@ -322,6 +322,26 @@ public void DefineScope_WithNoParameters_ThrowsException_WhenFormatString_HasNam
             Assert.Equal(expectedMessage, exception.Message);
         }
 
+        [Theory]
+        [MemberData(nameof(DefineMethodsData))]
+        public void DefineMessage_ThrowsException_WhenFormatString_IsNull(Func<LogLevel, EventId, string, Delegate> define)
+        {
+            // Act
+            var exception = Assert.Throws<ArgumentNullException>(
+                () => define.Invoke(LogLevel.Error, 0, null));
+        }
+
+        public static IEnumerable<object[]> DefineMethodsData => new[]
+        {
+            new object[] { (Func<LogLevel, EventId, string, Delegate>)LoggerMessage.Define },
+            new object[] { (Func<LogLevel, EventId, string, Delegate>)LoggerMessage.Define<string> },
+            new object[] { (Func<LogLevel, EventId, string, Delegate>)LoggerMessage.Define<string, string> },
+            new object[] { (Func<LogLevel, EventId, string, Delegate>)LoggerMessage.Define<string, string, string> },
+            new object[] { (Func<LogLevel, EventId, string, Delegate>)LoggerMessage.Define<string, string, string, string> },
+            new object[] { (Func<LogLevel, EventId, string, Delegate>)LoggerMessage.Define<string, string, string, string, string> },
+            new object[] { (Func<LogLevel, EventId, string, Delegate>)LoggerMessage.Define<string, string, string, string, string, string> }
```

Comment:
Does DefineScope also fall in this code path? if so, perhaps we could add their overloads to test here as well.

Reply:
Good catch, actually. Yeah, I agree with that we want to add tests for this, but I guess it's for another test method and `MemberData`, because the method signatures are different - `Define(LogLevel, EventId, string)` vs `DefineScope(string)`. I will update my PR with the changes.

Reply:
Sure, I will add. Anyway, thanks for merging my first PR 🥳